extension/observer/kafkatopicsobserver/config_test.go

```diff
@@ -38,7 +38,7 @@ func TestLoadConfig(t *testing.T) {
 				ProtocolVersion:                      "3.7.0",
 				Brokers:                              []string{"1.2.3.4:9092", "2.3.4.5:9092"},
 				TopicRegex:                           "^topic[0-9]$",
-				TopicsSyncInterval:                   5 * time.Second,
```

Comment:
Any particular reason to make this lower?

Reply:
To speed up TestCollectEndpointsAllConfigSettings, which currently takes 16 seconds to run, and is now ~instantaneous.